apps/include/pcl/apps/impl/dominant_plane_segmentation.hpp

```diff
@@ -475,154 +477,154 @@ pcl::apps::DominantPlaneSegmentation<PointType>::compute_fast (std::vector<Cloud
   {
     std::map<float, float>::iterator it;
 
-    for (int i = 0; i < int (binary_cloud->width); i++)
-    {
-      for (int j = 0; j < int (binary_cloud->height); j++)
-      {
-        if (binary_cloud->at (i, j).intensity != 0)
-        {
-          //check if this is a root label...
-          it = connected_labels.find (binary_cloud->at (i, j).intensity);
-          while (it != connected_labels.end ())
-          {
-            //the label is on the list, change pixel intensity until it has a root label
-            (*binary_cloud) (i, j).intensity = (*it).second;
-            it = connected_labels.find (binary_cloud->at (i, j).intensity);
+    for (int i = 0; i < int(binary_cloud->width); i++) {
+      for (int j = 0; j < int(binary_cloud->height); j++) {
+        if (binary_cloud->at(i, j).intensity != 0) {
+          // check if this is a root label...
+          it = connected_labels.find(binary_cloud->at(i, j).intensity);
+          while (it != connected_labels.end()) {
+            // the label is on the list, change pixel intensity until it has a root
```

Comment:
Reword if possible (px for pixel) to keep the one word on the same line or break it in 2 lines